### PR TITLE
Add recruitment page to /legal/data-privacy

### PR DESCRIPTION
**Pull request by sowasred2012**
## Done

- Added /legal/data-privacy/recruitment based on the [copy doc](https://docs.google.com/document/d/1D_irZPx4n67_Lhg-9P2-Iu9jZM9Va7n0/edit#)

## QA

- Visit https://ubuntu-com-11606.demos.haus/legal/data-privacy/recruitment
- See that it matches the copy doc

### CODE REVIEW

**Comment by pmahnke** [comment]
@sowasred2012 are you adding a link to /legal/data-privacy as well?

File: templates/legal/data-privacy/recruitment.html [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+      <h4 class="p-muted-heading">Last Updated: 18/02/2022</h4>
+      <hr style="margin-bottom: 2rem;" />
+
+      <h1>Canonical Recruitment Privacy Notice</h1>
+
+      <h2 id="introduction">1. Introduction</h2>
+
+      <p>This Recruitment Privacy Notice describes how Canonical Group Limited (“Canonical”) collects, uses and retains personal information about you, when you apply or take part in our recruitment process. As such, this policy applies to potential candidates for employment, or those who may take part in recruitment programmes at Canonical.</p>
+
+      <p>At Canonical, we consider your privacy to be extremely important to us. Further information regarding our fundamental principles and practices can be found in our <a href="https://ubuntu.com/legal/data-privacy#:~:text=Canonical%20collects%20personal%20information%20from%20you%20in%20a%20number%20of%20different%20ways.&text=We%20don't%20share%20your,or%20to%20protect%20our%20rights.">Privacy Policy</a>.</p>
```
Comment (mtruj013) [inline templates/legal/data-privacy/recruitment.html:19]:
```suggestion
      <p>At Canonical, we consider your privacy to be extremely important to us. Further information regarding our fundamental principles and practices can be found in our <a href="/legal/data-privacy#:~:text=Canonical%20collects%20personal%20information%20from%20you%20in%20a%20number%20of%20different%20ways.&text=We%20don't%20share%20your,or%20to%20protect%20our%20rights.">Privacy Policy</a>.</p>
```

File: templates/legal/data-privacy/recruitment.html [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+
+      <h2 id="exercising-your-rights">11. Exercising your rights</h2>
+
+      <p>To exercise any of these rights above, please contact us at <a href="mailto:dataprotection@canonical.com">dataprotection@canonical.com</a>, providing your name and contact details. Depending on your request, we may ask for further verification of your identity, or additional information in order to fulfil your request.</p>
+
+      <h2 id="right-to-complain">12. Right to complain</h2>
+
+      <p>If you wish to make a complaint to us about how we use or retain your personal information, please contact us at <a href="mailto:dataprotection@canonical.com">dataprotection@canonical.com</a>.</p>
+
+      <p>You also have the right to submit a complaint with the UK data protection authority, which is the Information Commissioner's Office (ICO). If you wish to file a complaint with the ICO, you may find the following information helpful: https://ico.org.uk/make-a-complaint/</p>
```
Comment (mtruj013) [inline templates/legal/data-privacy/recruitment.html:141]:
```suggestion
      <p>You also have the right to submit a complaint with the UK data protection authority, which is the Information Commissioner's Office (ICO). If you wish to file a complaint with the ICO, you may find the following information helpful: <a href="https://ico.org.uk/make-a-complaint/">https://ico.org.uk/make-a-complaint/<a></p>
```

**Review by mtruj013 — APPROVED** [review]
LGTM, just a couple small things

File: templates/legal/data-privacy/recruitment.html (hunk 1 above)
Comment (sowasred2012) [reply templates/legal/data-privacy/recruitment.html:19]: Ahhhh yes, nice, will commit these - thanks!

**Pull request merged**